server/src/models/people/populate.js

```diff
@@ -1,46 +1,184 @@
+    }))
+    .map(({ dept_code, ...values_by_year }) => ({
+      org_id: _.find(igoc_rows, (igoc_row) => igoc_row.dept_code === dept_code)
+        .org_id,
+      data: {
+        by_year: _.map(values_by_year, (value, year) => ({
+          year,
+          value,
+        })),
+      },
+    }))
+    .value();
+  const employee_age_totals = process_employee_data_sums(
+    "org_employee_age_group.csv"
+  );
+
+  const employee_ex_lvl_totals = process_employee_data_sums(
+    "org_employee_ex_lvl.csv"
   );
 
-  format_csv_records_then_register_to_headcount_models(
-    headcount_models_and_records
+  const employee_fol_totals = process_employee_data_sums(
+    "org_employee_fol.csv"
   );
+
+  const employee_gender_totals = process_employee_data_sums(
+    "org_employee_gender.csv"
+  );
+
+  const employee_region_totals = process_employee_data_sums(
+    "org_employee_region.csv"
+  );
+
+  const employee_type_totals = process_employee_data_sums(
+    "org_employee_type.csv"
+  );
+
+  const employee_gov_avgs = {
+    id: "gov",
+    data: _.chain(get_standard_csv_file_rows("org_employee_avg_age.csv"))
+      .filter({ dept_code: "ZGOC" })
+      .head()
+      .omit(["dept_code", "dimension"])
+      .map((value, key) => ({
+        by_year: {
+          year: [key][0],
+          value: value,
+        },
+      }))
+      .value(),
+  };
+
+  const org_id_list = _.map(employee_ex_lvl_rows, "org_id"); // all the csv files should have the same list of org_ids
+  const find_by_org_id = (rows, org_id) =>
+    _.find(rows, (row) => row.org_id === org_id).data;
+
+  const org_summary = _.map(org_id_list, (org_id) => ({
+    org_id: org_id,
+    employee_age_group: {
+      org_id: org_id,
+      data: find_by_org_id(employee_age_rows, org_id),
+    },
+    employee_ex_lvl: {
+      org_id: org_id,
+      data: find_by_org_id(employee_ex_lvl_rows, org_id),
+    },
+    employee_fol: {
+      org_id: org_id,
+      data: find_by_org_id(employee_fol_rows, org_id),
+    },
+    employee_gender: {
+      org_id: org_id,
+      data: find_by_org_id(employee_gender_rows, org_id),
+    },
+    employee_region: {
+      org_id: org_id,
+      data: find_by_org_id(employee_region_rows, org_id),
+    },
+    employee_type: {
+      org_id: org_id,
+      data: find_by_org_id(employee_type_rows, org_id),
+    },
+    employee_avg_age: {
+      org_id: org_id,
+      data: find_by_org_id(employee_avg_age_rows, org_id),
+    },
+  }));
+  const gov_summary = {
+    id: "gov",
+    employee_age_totals: employee_age_totals,
+    employee_ex_lvl_totals: employee_ex_lvl_totals,
+    employee_gender_totals: employee_gender_totals,
+    employee_fol_totals: employee_fol_totals,
+    employee_region_totals: employee_region_totals,
+    employee_type_totals: employee_type_totals,
+    employee_gov_avgs: employee_gov_avgs,
+  };
+  return await Promise.all([
+    OrgEmployeeSummary.insertMany(org_summary),
+    GovEmployeeSummary.insertMany(gov_summary),
```

Comment:
What about program summary?

Reply:
People data doesn't exist below the org level